src/dune_rules/setup.defaults.ml

```diff
@@ -11,7 +11,7 @@ let roots : string option Install.Roots.t =
   ; libexec_root = None
   }
 
+let prefix : string option = None
```

Comment:
Is it necessary to add the type here? I don't see where it is used in the code above.

Reply:
This seems to be an unrelated change that happens due to the changes from #11172. I've created a PR to commit it separately: #11198.